src/main/resources/logback.xml

```diff
@@ -5,19 +5,20 @@
             <pattern>%5relative %.-1level %20.20logger %msg %n</pattern>
         </encoder>
     </appender>
-    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>junit.log</file>
-        <append>false</append>
-        <encoder>
-            <!-- Reference at http://logback.qos.ch/manual/layouts.html#conversionWord -->
-            <pattern>%-4r %-5level %logger{35}: %msg%n</pattern>
-        </encoder>
-    </appender>
+<!-- Commented out because this causes a junit.log to be created whenever Alpha is run, even when no log output is written -->
```

Comment:
If there is configuration here that does not make sense, I´d rather it were removed instead of just commented out. Or is there a good reason to keep this stuff?

Reply:
I'd like to keep that for a bit yet - While the junit.log makes no sense for the distribution jar, it's not a bad idea for the tests. My plan would be to leave that as is for now and do the final version of the logback.xml when I take care of issue #204 where I plan to create separate xmls for tests and the distribution jar